proc-macros/src/rpc_macro.rs

```diff
@@ -26,16 +26,14 @@
 
 //! Declaration of the JSON RPC generator procedural macros.
 
-use crate::{
-	attributes::{
-		optional, parse_param_kind, Aliases, Argument, AttributeMeta, MissingArgument, NameMapping, ParamKind, Resource,
-	},
-	helpers::extract_doc_comments,
-};
+use std::borrow::Cow;
```

Comment:
use `beef::Cow`? :)

Reply:
It's a bit annoying with `beef::Cow` because it doesn't have `ToTokens` impl, and re-borrowing everything to `&str` on stack after call adds boilerplate.

Could add a feature flag adding `ToTokens` to `beef` maybe later.

Reply:
ah right it's macros, nvm